test/features/bridge/sqlserver/sqlserver_bridge_incremental.feature

```diff
@@ -28,33 +28,33 @@ Feature: [SQLS-BRG] Bridge table
       | md5('1002') | 2018-06-01 00:00:00.000 | md5('1002\|\|200')     |
       | md5('1003') | 2018-06-01 00:00:00.000 | md5('1003\|\|300')     |
       | md5('1004') | 2018-06-01 00:00:00.000 | md5('1004\|\|400')     |
-    Given the RAW_CUSTOMER_ORDER table contains data
-      | CUSTOMER_ID | ORDER_ID | LOAD_DATETIME           | END_DATE                | SOURCE |
-      | 1001        | 101      | 2018-06-01 12:00:00.000 | 9999-12-31 23:59:59.996 | *      |
-      | 1002        | 100      | 2018-06-01 12:00:00.000 | 9999-12-31 23:59:59.996 | *      |
-    And I stage the STG_CUSTOMER_ORDER data
-    And the AS_OF_DATE table is created and populated with data
-      | AS_OF_DATE              |
-      | 2018-06-01 00:00:00.000 |
-      | 2018-06-01 12:00:00.000 |
-      | 2018-06-02 00:00:00.000 |
-    When I load the vault
-    Then the BRIDGE_CUSTOMER_ORDER table should contain expected data
-      | CUSTOMER_PK | AS_OF_DATE              | LINK_CUSTOMER_ORDER_PK |
-      | md5('1001') | 2018-06-01 00:00:00.000 | md5('1001\|\|100')     |
-      | md5('1002') | 2018-06-01 00:00:00.000 | md5('1002\|\|200')     |
-      | md5('1003') | 2018-06-01 00:00:00.000 | md5('1003\|\|300')     |
-      | md5('1004') | 2018-06-01 00:00:00.000 | md5('1004\|\|400')     |
-      | md5('1001') | 2018-06-01 12:00:00.000 | md5('1001\|\|101')     |
-      | md5('1002') | 2018-06-01 12:00:00.000 | md5('1002\|\|100')     |
-      | md5('1002') | 2018-06-01 12:00:00.000 | md5('1002\|\|200')     |
-      | md5('1003') | 2018-06-01 12:00:00.000 | md5('1003\|\|300')     |
-      | md5('1004') | 2018-06-01 12:00:00.000 | md5('1004\|\|400')     |
-      | md5('1001') | 2018-06-02 00:00:00.000 | md5('1001\|\|101')     |
-      | md5('1002') | 2018-06-02 00:00:00.000 | md5('1002\|\|100')     |
-      | md5('1002') | 2018-06-02 00:00:00.000 | md5('1002\|\|200')     |
-      | md5('1003') | 2018-06-02 00:00:00.000 | md5('1003\|\|300')     |
-      | md5('1004') | 2018-06-02 00:00:00.000 | md5('1004\|\|400')     |
+#    Given the RAW_CUSTOMER_ORDER table contains data
```

Comment:
Is this intentionally commented out? Related to DATETIME2?

Reply:
No, this looks like the handiwork of my arch enemy the AntiTim. I have removed the comments and confirmed that the tests are passing.